### PR TITLE
[16.0][MIG] base_export_manager: Migration to version 16.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT43220

This Pr superseeds: #649 

@CarlosRoca13 @chienandalu please review!

### CODE REVIEW

File: base_export_manager/models/ir_exports_line.py [hunk 1]
```diff
@@ -142,7 +142,8 @@ def _inverse_name(self):
             if any(parts):
                 # invalidate_cache -> in order to get actual value of field 'label'
                 # in function '_check_name'
-                self.invalidate_cache(ids=one.ids)
+                # self.env["ir.exports.line"].invalidate_cache(ids=one.ids)
```
Comment (chienandalu) [inline base_export_manager/models/ir_exports_line.py]: Don't leave commented code
```suggestion
```

File: base_export_manager/models/ir_exports_line.py [hunk 2]
```diff
@@ -142,7 +142,8 @@ def _inverse_name(self):
             if any(parts):
                 # invalidate_cache -> in order to get actual value of field 'label'
```
Comment (chienandalu) [inline base_export_manager/models/ir_exports_line.py]: Update the comment

**Review by chienandalu — CHANGES_REQUESTED** [review]
Functionally looks good to me. Some minor code comments:

(also, some commits can be squashed)

**Comment by pedrobaeza** [comment]
Did you see #563 ?

**Comment by chienandalu** [comment]
This superseeds a more complete PR. Anyway, it has a better approach to the invalidate_recordset approach is maybe more clear and self explanatory

**Comment by pedrobaeza** [comment]
Then it's fair to credit the author of the first PR ever. What about rebasing this one over the other?

**Comment by chienandalu** [comment]
Ok, it shouldn't be that hard to cherry-pick and resolve a pair of conflicts.

**Comment by stefan-tecnativa** [comment]
> Ok, it shouldn't be that hard to cherry-pick and resolve a pair of conflicts.

I see the situation, what I do then? Rebase this PR with the other one? Or I cherry-pick the migration commit of the other PR, resolve the conflicts and, eventually, squash this PR's migration commit maintaining the authorship?

**Comment by pedrobaeza** [comment]
Save your current work, cherry pick the migration commit of the other PR, and put your relevant changes on it, doing another commit.

**Comment by chienandalu** [comment]
Well, cherry pick both migration commits indeed, as you've got two contributors to keep credit to

**Comment by stefan-tecnativa** [comment]
@chienandalu it's the history okay now? I will now review your comments

File: base_export_manager/views/ir_exports.xml [hunk 3]
```diff
@@ -29,52 +29,48 @@
         <field name="model">ir.exports</field>
         <field name="arch" type="xml">
             <form string="Export Profile">
-                <sheet>
```
Comment (chienandalu) [inline base_export_manager/views/ir_exports.xml]: You're dismissing the changes of the previous commit here.

Comment (stefan-tecnativa) [reply base_export_manager/views/ir_exports.xml]: Thanks for the advice, now it should be correct

**Review by kikopeiro — APPROVED** [review]
LGTM :+1 (without code review)

Thanks!

**Comment by pedrobaeza** [comment]
Please include #704

**Review by chienandalu — APPROVED** [review]
:+1:

**Pull request merged**